.github/workflows/create-pr.yml

```diff
@@ -0,0 +1,21 @@
+name: Open Pull Request
+
+on:
+  push:
+    branches:
+      - pulls/**
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - uses: ./
+        with:
+          helper: create-pr
+          title: New PR
+          target: main
+          body: Implemented new feature. Added tests.
+          github_token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
This will likely require more than just the default github token. I think you will need a personal access token, similar to other helpers

Reply:
Updated.